.clang-format

```diff
@@ -86,5 +86,14 @@ SpacesInCStyleCastParentheses: false
 SpacesInParentheses: false
 SpacesInSquareBrackets: false
 Standard: c++11
+StatementMacros:
```

Comment:
Those are the relevant changes.

It tells clang-format to treat the macro like a statement, so that it adds a linebreak without indentation afterwards